recipes/pystack/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pystack" %}
```

Comment:
There is a package on PyPI with the same name. Should I keep this name here?
I noted that when there is a conflict, the PyPI package is normally named ```python-*```